checkstyle_violated_rules.txt

```diff
@@ -0,0 +1 @@
+WARN
```

Comment:
This isn't a rule name. I'm not sure why you added this file, we don't need it either.